src/plugins/vis_type_timelion/public/helpers/panel_utils.ts

```diff
@@ -40,9 +38,27 @@ export interface Axis {
   tickLength: number;
   timezone: string;
   tickDecimals?: number;
-  tickFormatter: ((val: number) => string) | ((val: number, axis: Axis) => string);
+  tickFormatter: ((val: number) => string) | ((val: number, axis: any) => string);
   tickGenerator?(axis: Axis): number[];
   units?: { type: string };
+  domain?: {
+    min?: number;
+    max?: number;
+  };
+  position: 'top' | 'bottom' | 'left' | 'right' | undefined;
```

Comment:
We can use Position from elastc-charts here right?

Reply:
done